app/allocations/middleware/set-body.allocations.js

```diff
@@ -6,6 +6,8 @@ function setBodyAllocations(req, res, next) {
 
   set(req, 'body.allocations', {
     status,
+    sortBy: req.query.sortBy,
```

Comment:
These values could be destructured above like `status`

Reply:
I wonder if using the sort as an object may also make it neater to work with and override in the allocations to table presenter too?

So the URL would be:

```javascript
?foo=bar&sort[by]=field&sort[direction]=asc
```

and you just pass the sort object on the body:

```javascript
const { status, sort } = req.query

set(req, 'body.allocations', {
  status,
  sort,
  ...
})
```

Reply:
Yeah, I toyed with the idea of a tuple-like object. Or just an object. It's a possibility.